redcap_bridge/test_redcap/test_cli.py

```diff
@@ -30,23 +30,23 @@ def test_download(initialize_test_dir):
     output_file = test_directory / 'cli_download_test.csv'
 
     # download with default arguments
-    result = subprocess.run(['RedCapBridge', 'download', output_file, SERVER_CONFIG_YAML],
+    result = subprocess.run(['RedCapBridge', 'download', output_file, SERVER_CONFIG_YAML, '--server', 'redcap'],
                             stdout=subprocess.PIPE)
     assert 'error' not in str(result.stdout)
     assert output_file.exists()
     output_file.unlink()
 
     # download in compressed mode
     result = subprocess.run(['RedCapBridge', 'download', '--compressed', output_file,
-                             SERVER_CONFIG_YAML],
+                             SERVER_CONFIG_YAML, '--server', 'redcap'],
```

Comment:
@killianrochet Could you keep the redcap-cli tests as they are and just add more tests to cover the elab cases?